contracts/src/v0.8/automation/chains/ScrollModule.sol

```diff
@@ -3,8 +3,12 @@ pragma solidity 0.8.19;
 
 import {IScrollL1GasPriceOracle} from "../../vendor/@scroll-tech/contracts/src/L2/predeploys/IScrollL1GasPriceOracle.sol";
 import {ChainModuleBase} from "./ChainModuleBase.sol";
+import {ConfirmedOwner} from "../../shared/access/ConfirmedOwner.sol";
+
+contract ScrollModule is ChainModuleBase, ConfirmedOwner {
```

Comment:
We now have an owner of this chain module, we'll need to add appropriate steps in deployment and setup guide cc @anirudhwarrier